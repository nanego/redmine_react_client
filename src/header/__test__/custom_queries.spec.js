import React from 'react';
import CustomQueries from '../custom_queries';
import renderer from 'react-test-renderer';

it('should render CustomQueries', () => {
  var replaceSelectedFilters = function(){};
  const component = renderer.create(
      <CustomQueries replaceSelectedFilters={replaceSelectedFilters} />
  );
  let select = component.toJSON();
  expect(select).toMatchSnapshot();
});