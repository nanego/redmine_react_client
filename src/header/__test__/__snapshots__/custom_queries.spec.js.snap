exports[`test should render CustomQueries 1`] = `
<div
  className="ui secondary vertical menu">
  <a
    className="item"
    onClick={[Function]}>
    Mes demandes par priorité
  </a>
  <a
    className="item"
    onClick={[Function]}>
    Demandes surveillées
  </a>
  <a
    className="item"
    onClick={[Function]}>
    Traité sans activité récente
  </a>
  <a
    className="item"
    onClick={[Function]}>
    Filtres du permanent
  </a>
  <div
    className="ui divider" />
  <div
    className="item"
    onClick={[Function]}>
    Recherche avancée
  </div>
</div>
`;
